mgmt/src/models/internal/natconfig/table_extend.rs

```diff
@@ -138,6 +138,14 @@ fn collapse_prefix_lists(
     excludes: &BTreeSet<Prefix>,
 ) -> Result<BTreeSet<Prefix>, NatPeeringError> {
     let mut result = prefixes.clone();
+
+    if result.is_empty() && !excludes.is_empty() {
```

Comment:
Did we decide if these configs are valid or if we want to reject them?

Reply:
Right now we reject them.

I need to clear with @Frostman the exact semantics of empty `ips`/`as` and non-empty `nots`/`not_as` to be sure to validate the `VpcExpose` correctly. But it's not at the top of my priority list at the moment.